### PR TITLE
add zxpinstaller 1.5.0

### LINKED ISSUE #24731: Cask request: ZXPInstaller version 1.5.0
### Cask details

**Name**
ZXPInstaller

**Homepage**
http://zxpinstaller.com/

**License**
GNU - [https://github.com/CreativeDo/ZXPInstaller/blob/master/LICENSE](https://github.com/CreativeDo/ZXPInstaller/blob/master/LICENSE)

**Download URL**
https://github.com/CreativeDo/ZXPInstaller/releases/download/1.5.0/ZXPInstaller.dmg 

**Description**
ZXPInstaller is a simple cross platform (OS X and Windows) installer for third party Adobe extensions (.zxp files). It serves as a replacement for the Extension Manager which no longer is not supported after CC 2014.

### PR DESCRIPTION
**Pull request by numbermaniac**
### Checklist
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

Additionally, when **adding a new cask**:
- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

Closes #24731.

### CODE REVIEW

File: Casks/zxpinstaller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'zxpinstaller' do
+  version '1.5.0'
+  sha256 'fe54a02b891c99d4452f723447166e2d1da0c3a47a3ebe7b9762ed2243963ba4'
+
+  # github.com/CreativeDo was verified as official when first introduced to the cask
```
Comment (adidalal) [inline Casks/zxpinstaller.rb]: Needs a `# github.com/CreativeDo/ZXPInstaller was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

File: Casks/zxpinstaller.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  url "https://github.com/CreativeDo/ZXPInstaller/releases/download/#{version}/ZXPInstaller.dmg"
```
Comment (adidalal) [inline Casks/zxpinstaller.rb]: Mind adding an [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)?

**Review by adidalal — CHANGES_REQUESTED** [review]

File: Casks/zxpinstaller.rb (hunk 1 above)
Comment (Numbermaniac) [inline Casks/zxpinstaller.rb]: Done.

File: Casks/zxpinstaller.rb (hunk 2 above)
Comment (Numbermaniac) [inline Casks/zxpinstaller.rb]: Done.

**Review by adidalal — APPROVED** [review]

**Pull request merged**